libs/ledger-live-common/src/families/bitcoin/bridge/js.ts

```diff
@@ -35,29 +33,10 @@ const updateTransaction = (t, patch): any => {
   return updatedT;
 };
 
-const preload = async (): Promise<{
-  explorerConfig: FullConfigOverrides | null | undefined;
-}> => {
-  const explorerConfig = await explorerConfigAPI.preload();
-  return {
-    explorerConfig,
-  };
-};
-
-const hydrate = (maybeConfig: any): void => {
-  if (
-    typeof maybeConfig === "object" &&
-    maybeConfig &&
-    maybeConfig.explorerConfig
-  ) {
-    explorerConfigAPI.hydrate(maybeConfig.explorerConfig);
-  }
-};
-
 const currencyBridge: CurrencyBridge = {
   scanAccounts,
-  preload,
-  hydrate,
+  preload: () => Promise.resolve({}),
+  hydrate: () => {},
```

Comment:
these methods are optional, you can remove their def i think

Reply:
I tried and vscode is not happy

Reply:
![image](https://user-images.githubusercontent.com/19901996/211340792-4c744a05-bde0-403f-8007-6052823a015f.png)

Doesn't look optional ! In the type atleast